src/dune_rules/dune_file.mli

```diff
@@ -362,6 +373,7 @@ type Stanza.t +=
   | Tests of Tests.t
   | Include_subdirs of Loc.t * Include_subdirs.t
   | Toplevel of Toplevel.t
+  | Library_redirect of Library_redirect.Private.t
```

Comment:
Adding a separate stanza for this means that the pattern matches that match on `Deprecated_library_name` no longer apply to `Library_redirect`. Is this deliberate?

Reply:
Never mind, this is clearly deliberate because. I should read before I speak.